proxy/src/main/java/com/velocitypowered/proxy/tablist/KeyedVelocityTabList.java

```diff
@@ -90,6 +90,20 @@ public void addEntry(TabListEntry entry) {
     entries.put(entry.getProfile().getId(), (KeyedVelocityTabListEntry) entry);
   }
 
+  @Override
+  public void addEntries(Collection<TabListEntry> entries) {
+    for (TabListEntry entry : entries) {
+      addEntry(entry);
+    }
+  }
+
+  @Override
+  public void addEntries(TabListEntry... entries) {
+    for (TabListEntry entry : entries) {
+      addEntry(entry);
+    }
+  }
```

Comment:
Instead of making an `@Override` in each implementation, it would be better to implement these methods as a default method in the TabList interface